lib/tori/backend/s3.rb

```diff
@@ -91,6 +91,17 @@ def read(filename)
         body(filename).read
       end
 
+      def public_url(filename)
+        "#{client.config.endpoint}/#{@bucket}/#{filename}"
+      end
+
+      def url_for(filename, method)
+        signer = Aws::S3::Presigner.new(client: client)
+        signer.presigned_url(method, bucket: @bucket, key: filename)
+      end
+
+      private
+
```

Comment:
I've changed like this fix in the past.
But I decided to public all method https://github.com/ksss/tori/commit/52870236dd8f733bb739eb37e08db30b8a38c39b